zerver/views/users.py

```diff
@@ -49,6 +50,7 @@
     check_valid_bot_type,
     check_valid_interface_type,
     get_api_key,
+    get_bots_by_owner,
```

Comment:
These two imports look out of place in this commit. It looks like that they should belong to the previous one.

Reply:
Thanks I'll look into this.